pro/src/core/OfferEducational/types.ts

```diff
@@ -164,6 +164,7 @@ export type CollectiveStockResponseModel = {
   isEducationalStockEditable?: boolean
   educationalPriceDetail?: string
   stockId?: string
+  offerId: string // FIXME (MathildeDuboille, 22-04-2022): offerId should not exist on CollectiveStockResponseModel, but as it exists on StockResponse ts crashes if I remove this line
```

Comment:
if this line is removed*

Peut-être indiquer quand est-ce qu'on pourra l'enlever aussi ?
Remove once new collective model is in production → Sinon mettre dans le ticket de clean qu'il faut enlever ça ?